Core/HLE/sceHttp.cpp

```diff
@@ -588,6 +588,8 @@ static int sceHttpCreateConnection(int templateID, const char *hostString, const
 	if (httpObjects[templateID - 1LL]->className() != name_HTTPTemplate)
 		return hleLogError(Log::sceNet, SCE_HTTP_ERROR_INVALID_ID, "invalid id");
 
+	// TODO: Look up hostString in DNS here.
```

Comment:
As i remembered the `hostString` contains a full path, thus resolving here instead of the resolver at HTTPClient.cpp might not works properly on shared webhosting that use subdomain, since each subdomains might be pointed to the same IP and the server might routing it to the correct server based on the subdomain string.

Reply:
Hm, good point...